Client/mods/deathmatch/logic/CResourceModelStreamer.h

```diff
@@ -26,5 +26,5 @@ class CResourceModelStreamer
     void FullyReleaseModel(std::uint16_t modelId);
 
 private:
-    std::unordered_map<std::uint16_t, std::uint16_t> m_requestedModels;
+    std::map<std::uint16_t, std::uint16_t> m_requestedModels;
```

Comment:
Could you elaborate on how exactly `std::unordered_map` was involved in the crash? And why does `std::map` fix it? I just want to understand the logic.

Reply:
You right. This fix doesn't changes anything. CClientModelManager stores a pointer to a deleted CResource object. CClientModel::Deallocate uses dead pointer